### ISSUE TITLE
KeyError: 'StatementType' with Athena using max_cache_seconds

### ISSUE THREAD

**Issue by zacharycarter** [awslabs/aws-data-wrangler#323]
**Describe the bug**
I'm trying to execute a query against AWS Athena - 

```
pricing_table = awswrangler.athena.read_sql_query(
        (
            "SELECT * FROM historical_lane_pricing "
            "WHERE postal_code in ({}) "
            "AND company_id = '{}'"
        ).format(
            ', '.join(["'{}'".format(stop_zip) for stop_zip in stops_zips]),
            pricing_request.company_id,
        ),
        database = "default",
        max_cache_seconds=900,
        max_cache_query_inspections=500
    )
```

Which is producing the falling terror - 
```
KeyError: 'StatementType'
```

**To Reproduce**
This doesn't happen against master branch, only with PyPI installed v 1.6.3

Run a query against AWS Athena using the `read_sql_query` method and setting a value > 0 for the `max_cache_seconds` argument.

**Comment by igorborgest** [comment]
Thank you for reporting it @zacharycarter!

Could you please check your boto3 and botocore versions in both situation (master vs pypi)?

In the meanwhile I will try to replicate it here.

**Comment by igorborgest** [comment]
Fix done on PR #325. Release expected for version `1.7.0` next week.

**Comment by igorborgest** [comment]
@zacharycarter, feel free to test our dev branch before the official release:

```
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by zacharycarter** [comment]
Hi @igorborgest - please accept my apologies for not getting back to you on boto3 / botocore versions, and thank you for reproducing and addressing this so quickly! We had a hackathon going on at work today so I was preoccupied with that.

I will test this tomorrow against the dev branch and confirm that everything is working for me.

**Comment by zacharycarter** [comment]
@igorborgest just wanted to confirm I've tested against the dev branch and all looks good! Thanks again for the quick fix!

**Comment by igorborgest** [comment]
Awesome @zacharycarter, thank you for test it.

**Comment by igorborgest** [comment]
Released in [1.7.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.7.0)!

**Comment by igorborgest** [comment]
Hi @zacharycarter !

It is totally off-topic, but we are stating a [_"Who uses AWS Data Wrangler?"_](https://github.com/awslabs/aws-data-wrangler/tree/dev#who-uses-aws-data-wrangler) section. So feel free to add yourself if you want 😄 .
